.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/alpine/.devcontainer/base.Dockerfile
```

Comment:
What is this file? If something for local development, remove and add to gitignore

Reply:
sorry about that, will remove.

Reply:
done